### DISCUSSION TITLE
About Apple Live Photo

### DISCUSSION THREAD

**Discussion by HenKKer** [immich-app/immich#14052] [Feature Request]
May I kindly ask if Immich plans to add support for Apple Live Photos in future versions?

**Comment by mmomjian** [comment]
Live photos on iOS are fully supported by Immich. Is there something you have found missing?

**Reply by HenKKer** [reply]
May I kindly ask why my Live Photo is being converted to the .HEIC format after uploading? I initially thought it wasn't supported.

**Reply by HenKKer** [reply]
Oh! I understand the reason now and have successfully resolved the issue. Thank you for your reply.
